lib/sensu-handler.rb

```diff
@@ -110,6 +110,24 @@ def api_request(method, path, &blk)
       res
     end
 
+   def stash_request(method, path, data = '', &blk)
+     if api_settings.nil?
+       raise "api.json settings not found."
+     end
+     domain = api_settings['host'].start_with?('http') ? api_settings['host'] : 'http://' + api_settings['host']
+     uri = URI("#{domain}:#{api_settings['port']}#{path}")
```

Comment:
Would you consider isolating these lines which construct the API URI into a method for generating so that both the api_request and stash_request methods can use it uniformly?